src/cbc_sdk/platform/alerts.py

```diff
@@ -1574,6 +1574,22 @@ def set_group_by(self, field):
         self._group_by = field
         return self
 
+    def _build_request(self, from_row, max_rows, add_sort=True):
```

Comment:
We already do this in `_perform_query` so if we are going to define this function we need to update the following

```
while still_querying:
    request = self._build_request(current, max_rows)
    request["group_by"] = {"field": self._group_by}
```

Reply:
removed from _perform_query